setup.py

```diff
@@ -112,9 +85,7 @@ def run(self):
     },
 
     install_requires=[
-        # Any changes here must be reflected in ``python-flocker.spec.in`` so
-        # that RPM dependencies match.
-        # They must also be reflected in the yumdownloader lines in
+        # Any changes here must be reflected in the yumdownloader lines in
```

Comment:
Okay I don't really know what to do here.  It seems like we could drop the whole `Pre-populating RPM Repository` section right now and not be any worse off.  Perhaps at the cost of having to re-introduce these steps later if we gain a non-standard dependency.  We've managed to avoid that so far, though, it seems...?